src/pages/index.js

```diff
@@ -64,8 +158,49 @@ document.querySelector(".profile__button-add").addEventListener("click", () => {
 });
 
 function createCard(cardData) {
-  const card = new Card(cardData, "#card", () => {
-    pictureModal.open(card.getCardData());
-  });
-  return card.getCardElement();
+  const card = new Card(
+    cardData,
+    userId,
+    "#card",
+    () => {
+      pictureModal.open(card.getCardData());
+    },
+    (cardId) => {
+      deleteModal.open();
+      deleteModal.setSubmitHandler(() => {
+        deleteModal.changeButtonText("Deleting...");
+        api
+          .deleteCard(cardId)
+          .then(() => {
+            card.deleteCard();
+          })
+          .catch((err) => {
+            console.error("Error. The request has failed: ", err);
+          })
+          .finally(() => {
+            deleteModal.changeButtonText("Yes");
+            deleteModal.close();
+          });
+      });
+    },
+    () => {
+      api
+        .toggleCardLike(card.getCardData().id, card.getCardData().isLiked)
+        .then((res) => {
+          {
+            card.setIsLiked(res.likes);
+            card.toggleActiveLike();
+            card.updateLikeAmount(res.likes);
+          }
+        });
```

Comment:
You need to `catch` possible errors at the very end of any server request.

**Please, correct it everywhere**